packages/api/app/Validators/GameValidator.ts

```diff
@@ -1,12 +1,13 @@
 import { HttpContextContract } from '@ioc:Adonis/Core/HttpContext';
 import { schema, rules } from '@ioc:Adonis/Core/Validator';
+import { DifficultyEnum } from 'App/Enums/Difficulty';
 
 export default class GameValidator {
   constructor(private ctx: HttpContextContract) {}
 
   public schema = schema.create({
     title: schema.string({ trim: true }, [rules.maxLength(80)]),
-    difficultyId: schema.number([rules.exists({ table: 'difficulties', column: 'id' })]),
+    difficultyId: schema.number([rules.range(0, Object.keys(DifficultyEnum).length / 2)]),
```

Comment:
I don't really know why you divide by 2 ? You should only do -1 ?
Also don't forget to change the message `difficultyId.exists` -> `difficultyId.range`

Reply:
Yeah the `/ 2` is a mistake